jcl-custom-rules/src/main/java/com/sonarsource/jcl/example/MyJclCustomRulesDefinition.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2024-2024 SonarSource SA
+ * All rights reserved
+ * mailto:info AT sonarsource DOT com
+ */
+package com.sonarsource.jcl.example;
+
+import org.sonar.plugins.jcl.api.CustomRulesDefinition;
+import com.sonarsource.jcl.example.checks.MyCustomRule;
+import java.util.List;
+import org.sonar.api.server.rule.RulesDefinition;
+
+public class MyJclCustomRulesDefinition implements CustomRulesDefinition {
+
+  public static final String REPOSITORY_KEY = "MyCustomJclRules";
+
+  @Override
+  public void define(RulesDefinition.Context context) {
```

Comment:
Minor remark: The corresponding code in `sonar-jcl` doesn't have `RulesDefinition.`, which is imported above.
So I think `RulesDefinition.` is unnecessary and can be removed.